### PR TITLE
web: move global styles to `wildcard`

### PR DESCRIPTION
**Pull request by valerybugakov**
## Context

1. Moves branded styles to the `wildcard` package because `wildcard` components still rely on global styles. This removes all circular imports between the `wildcard` and `branded` packages.
2. Introduces a new `wildcard` entry point `@sourcegraph/wildcard/src/stories` used to export Storybook utilities needed in other packages.
3. Consolidates all branded styles in the `wildcard` package. Parts of `web` and `branded` global styles are moved into the `wildcard`. 
4. Removes unused global styles.

## Test plan

1. CI
2. There're no `branded` imports in the `wildcard` package.
3. There're no `branded` imports in the `shared` package.

## App preview:

- [Web](https://sg-web-vb-move-global-styles-to-wildcard.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| -0.01% (-0.16 kb) | -0.00% (-0.16 kb) | 0.00% (0.00 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/vb/move-global-styles-to-wildcard/compare-report.html;tab=live_object) for a full comparison between the commits 05cace5 and 0a926a5 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: client/branded/src/components/panel/TabbedPanelContent.story.tsx [hunk 1]
```diff
@@ -5,15 +5,13 @@ import { EMPTY, of } from 'rxjs'
 import { FlatExtensionHostAPI } from '@sourcegraph/shared/src/api/contract'
 import { pretendProxySubscribable, pretendRemote } from '@sourcegraph/shared/src/api/util'
 import { extensionsController } from '@sourcegraph/shared/src/testing/searchTestHelpers'
-import webStyles from '@sourcegraph/web/src/SourcegraphWebApp.scss'
```
Comment (vovakulikov) [inline client/branded/src/components/panel/TabbedPanelContent.story.tsx:8]: 👍

File: client/web/src/SourcegraphWebApp.scss [hunk 2]
```diff
@@ -5,19 +5,11 @@ It should import all component stylesheets.
 
 // Global libraries styles
 @import 'react-grid-layout/css/styles.css';
-@import 'react-resizable/css/styles.css';
```
Comment (vovakulikov) [inline client/web/src/SourcegraphWebApp.scss:8]: I think we should move react-grid-layout and react-resizable to the insight specific styles chunk but before this I think removing `react-resizable` will introduce visual regression for the insight dashboard cards

File: client/wildcard/src/global-styles/base.scss [hunk 3]
```diff
@@ -40,3 +40,23 @@ input[type='checkbox'] {
         cursor: pointer;
     }
 }
+
+@import 'react-resizable/css/styles.css';
```
Comment (vovakulikov) [inline client/wildcard/src/global-styles/base.scss:44]: Oh I see it's here now. But why react resizable only?

**Review by vovakulikov — APPROVED** [review]

File: client/wildcard/src/global-styles/base.scss (hunk 3 above)
Comment (valerybugakov) [reply client/wildcard/src/global-styles/base.scss:44]: It looks like `react-grid-layout` is not used in wildcard components and is specific to the web application, so I left it in the web application stylesheet.

Comment (vovakulikov) [reply client/wildcard/src/global-styles/base.scss:44]: is `react-resizable` used somewhere in the wildcard?

Comment (valerybugakov) [reply client/wildcard/src/global-styles/base.scss:44]: Yep, in chart stories.

Comment (vovakulikov) [reply client/wildcard/src/global-styles/base.scss:44]: if it's used only in storybook stories should we not include this styles in the global wildcard chunk?

**Pull request merged**